modules/site-icon/jetpack-site-icon.php

```diff
@@ -96,7 +96,13 @@ public function jetpack_modules_loaded() {
 	 *
 	 */
 	public function site_icon_add_meta() {
-
+		/**
+		 * Toggles the Favicon meta elements from being loaded.
+		 *
+		 * @since 3.2.0
+		 *
+		 * @param bool false Default is false, keep at false if you want to never load this option.
```

Comment:
Could you explain this differently, drop the first false, and reorder?
`@param bool [What does this do?] Default is false.`

"Load this option" isn't clear to me. "Sets the display of the favicon" or something like that explains it better, IMO.